Tasks/DeployVisualStudioTestAgent/InstallTestAgent.ps1

```diff
@@ -83,20 +76,32 @@ function Install-Product($SetupPath, $ProductVersion, $Update)
 
 	try
 	{
+        # Invoke the TA installation
 		if($ProductVersion -eq "15.0") {
 			$tpPath = Join-Path $env:SystemDrive TestAgent2017
-			$argumentsarr = @("--noRestart","--quiet","--installPath","$tpPath")
+			$exitCode = InstallTestAgent2017 -SetupPath $SetupPath -InstallPath $tpPath
```

Comment:
+1. Aren't we checking why the install success ?

---
In reply to: [101449538](https://github.com/Microsoft/vsts-tasks/pull/3623#discussion_r101449538) [](ancestors = 101449538)